contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -642,10 +642,18 @@
         return (this.firstNode && this.getContentNodeFiles(this.firstNode.id)) || [];
       },
       fileDuration() {
+        let audioVideoFiles;
         if (this.firstNode.kind === 'audio' || this.firstNode.kind === 'video') {
-          return this.nodeFiles.filter(
+          audioVideoFiles = this.nodeFiles.filter(
             file => file.file_format === 'mp4' || file.file_format === 'mp3'
```

Comment:
I know you didn't change this, but it looks worthwhile to fix-- it's excluding the newly added `.webm` format. To go one step further, for posterity, I would suggest using the [presets information](https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/shared/leUtils/FormatPresets.js#L223) on what formats are video or audio resources

Reply:
I actually wonder if it's necessary to include this check here, now that I revisit it, since I believe the [presets are already checked earlier in upload](https://github.com/learningequality/studio/blob/unstable/contentcuration/contentcuration/frontend/shared/views/files/Uploader.vue#L136)?

Reply:
@bjester I've removed for now, based on my understanding of what's happening in upload, but if you think keeping this check (and updating to include `.webm`, using presets) here is helpful, I will add in a modified version

Reply:
Yeah I thought the same about keeping the check, and what I guessed was that since this uses `nodeFiles`, that it could run into subtitle files too?

Reply:
great point! I've updated @bjester